CMakeLists.txt

```diff
@@ -638,12 +642,18 @@ if(NOT AVIF_CODEC_AOM
     message(WARNING "libavif: No decoding library is enabled.")
 endif()
 
+add_library(avif_unicode OBJECT third_party/libwebp/unicode.c third_party/libwebp/unicode.h)
+target_include_directories(avif_unicode INTERFACE third_party/libwebp)
+if(AVIF_ENABLE_UNICODE)
```

Comment:
Instead of adding a WIN32-only build option `AVIF_ENABLE_UNICODE`, I wonder if we should test `WIN32` or `MSVC` here.